ignite/contrib/handlers/base_logger.py

```diff
@@ -191,7 +191,7 @@ def __init__(
         if not callable(reduction):
             raise TypeError(f"Argument reduction should be callable, but given {type(reduction)}")
 
-        def _is_0D_tensor(t: torch.Tensor) -> bool:
+        def _is_0D_tensor(t: Any) -> bool:
```

Comment:
Why this change ?

Reply:
I've got to change _is_0D_tensor parameter type hint as well. It wants to check if its argument is a tensor and beside that a 1D tensor so being Any is a reasonable assumption.

Reply:
Otherwise Mypy raised error.
```bash
error: Argument 1 to "_is_0D_tensor" has incompatible type
"Union[float, Tensor]"; expected "Tensor"  [arg-type]
```

Reply:
OK, it is actually correct that previous type hint is wrong. Thanks